tests/TASVideos.Test/Pages/UserFiles/InfoModelTests.cs

```diff
@@ -8,7 +8,7 @@
 namespace TASVideos.RazorPages.Tests.Pages.UserFiles;
 
 [TestClass]
-internal class InfoModelTests
+public sealed class InfoModelTests
```

Comment:
Why make this one random class sealed?

Reply:
I do that out of habit for Hawk, since it makes reflection slightly faster.
Key here is I'm making it `public` to resolve a warning which is from an Analyzer that was previously not active. I'll just revert all this and leave the warning since it seems that's what's causing the CI build to fail.